analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -174,6 +174,8 @@ def construct_analyzer_cmd(self, result_handler):
 
             analyzer_cmd.append(self.source_file)
 
+            analyzer_cmd.extend(['--export-fixes', result_handler.fixit_file])
```

Comment:
There is no difference in performance. Disk usage is a bigger problem, because the fixit directory is 11M with default profile and 27M with sensitive profile in case of Xerces. In the future we may consider binding this feature to a flag if it causes problem to the users.

Reply:
And I wouldn't bind it to the existence of `clang-apply-replacements` binary because many users or ours are using a different Clang during analysis which may not deploy this binary.